extensions/llama/python/llama.py

```diff
@@ -48,15 +45,30 @@ async def run_inference(
     ) -> ExecuteResult:
         resolved = resolve_prompt(prompt, parameters, aiconfig)
```

Comment:
Should add the logging for start/end events in the methods as well

Reply:
via callbacks?

Reply:
Yep